keyboards/40percentclub/sixpack/config.h

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright 2020
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x3430 // "40"
```

Comment:
This VID is not in use, but it is very close to what USB IF is issuing for new IDs.   I would prefer that this was something higher, to prevent collisions in the near future.

Reply:
VID was selected to align with the other 40percentclub boards using it (or FEED/CEEB), and per VIA recommendations.

`25/config.h:#define VENDOR_ID       0xFEED`
`4pack/config.h:#define VENDOR_ID 0xFEED`
`4x4/config.h:#define VENDOR_ID       0xFEED`
`5x5/config.h:#define VENDOR_ID       0xFEED`
`6lit/config.h:#define VENDOR_ID       0xFEED`
`foobar/config.h:#define VENDOR_ID       0xFEED`
`gherkin/config.h:#define VENDOR_ID       0xFEED`  ([#11040](https://github.com/qmk/qmk_firmware/pull/11040#issuecomment-735672452))
`half_n_half/config.h:#define VENDOR_ID       0xFEED`
`i75/config.h:#define VENDOR_ID       0xFEED`
`luddite/config.h:#define VENDOR_ID       0x3430 // "40"`
`mf68/config.h:#define VENDOR_ID       0xCEEB`
`nano/config.h:#define VENDOR_ID       0xCEEB`
`nein/config.h:#define VENDOR_ID       0x3430 // "40"`
`nori/config.h:#define VENDOR_ID       0xFEED`
`sixpack/config.h:#define VENDOR_ID       0x3430 // "40"`
`tomato/config.h:#define VENDOR_ID       0xFEED`
`ut47/config.h:#define VENDOR_ID       0xFEED`

Reply:
The VIA docs specifically recommend changing the VID: https://caniusevia.com/docs/configuring_qmk/#change-vendor_id-product_id

Reply:
I mean, I do understand that, but I didn't catch this on the other ones.

For instance, ZSA (ergodox ez, moonlander) was assigned 0x3297 by USB-IF last year.  And 0x3339 has already been assigned (and appears to be the last assigned).  That means that 0x3430 may/will be assigned in the very near future, hence my concern about this low of a number.

I would personally prefer 0xCEEB over 0x3430.

Reply:
Given it exists in the repo already, I think it best to align for now. And bulk change all the existing ones if we still see it as an issue.